### PR TITLE
Add docker container for generate protoc, mock, goyacc and regel

### PR DESCRIPTION
**Pull request by CBists**
To generate packages with the same versions for all users

### CODE REVIEW

**Comment by reshke** [comment]
nice

File: docker-compose.yaml [hunk 1]
```diff
@@ -87,3 +87,7 @@ services:
         build:
             dockerfile: ./docker/slicer/Dockerfile
             context: .
+    generator:
+        build:
+            dockerfile: ./docker/generator/Dockerfile
+            context: .
```
Comment (EinKrebs) [inline docker-compose.yaml:93]: It's better not to include generator here, for it has no purpose. Better to use Docker directly in Makefile

**Review by EinKrebs — CHANGES_REQUESTED** [review]

File: docker/generator/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+FROM spqr-base-image
```
Comment (EinKrebs) [inline docker/generator/Dockerfile:1]: Maybe use `golang` docker image here, not to install go manually

File: Makefile [hunk 3]
```diff
@@ -159,10 +159,10 @@ yaccgen:
 gen: gogen yaccgen mockgen
 
 generate: build_images
-	docker compose up --remove-orphans generator
+	docker build -f docker/generator/Dockerfile -t spqr-generator .
+	docker run --name spqr-generator-1 spqr-generator --rm
```
Comment (EinKrebs) [inline Makefile:163]: You need to remove `--rm` flag

**Review by EinKrebs — CHANGES_REQUESTED** [review]

File: Makefile (hunk 3 above)
Comment (EinKrebs) [reply Makefile:163]: Also add `docker container rm spqr-generator-1` to the end of `generate` target

**Review by EinKrebs — APPROVED** [review]

**Pull request closed without merging**